src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs

```diff
@@ -29,17 +30,83 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     /// </summary>
     public partial class GeneralOptionsDialogControl : UserControl
     {
+        private ISonarLintSettings settings;
+        private ISonarLintDaemon daemon;
+
         public GeneralOptionsDialogControl()
         {
             InitializeComponent();
         }
 
-        private void OnInstallJavaScriptClicked(object sender, RoutedEventArgs e)
+        protected override void OnInitialized(EventArgs e)
+        {
+            base.OnInitialized(e);
+
+            UpdateActivateMoreButtonText();
+        }
+
+        private void UpdateActivateMoreButtonText()
+        {
+            string label;
```

Comment:
I would remove the local variable and directly set the content of the property.
Or use this:
```
ActivateMoreButton.Content = Settings.IsActivateMoreEnabled
    ? "Deactivate JavaScript support"
    : "Install and activate JavaScript support";
```